src/main/java/util/validator/Validator.java

```diff
@@ -0,0 +1,18 @@
+package util.validator;
+
+public class Validator {
+	private static final String NULL_MESSAGE = "NULL은 입력할 수 없습니다.";
+	private static final String EMPTY_MESSAGE = "빈 문자열은 입력할 수 없습니다.";
+
+	public static void isNull(String input){
+		if(input.equals(null)){
```

Comment:
```suggestion
		if(input == null){
```
equals와 ==의 차이에 대해 알아두면 좋을 것 같아요 :3